recipes/sentry-native/all/conandata.yml

```diff
@@ -21,17 +21,25 @@ patches:
   "0.4.10":
     - patch_file: "patches/0.4.10-0001-find-conan-qt.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/0.4.10-0002-CXX-14.patch"
```

Comment:
The actual patch is always the same, across different versions.
I did this to be consistent with the qt patch, but there is another PR removing said patch.
Should I use a single, version-agnostic, file?
It would be cleaner now but a future version may not take this patch. I don't know what are thw best practices here.

Reply:
Imho, it's fine to combine patches.
Me myself, I prefix the patch with the lowest version for which it is meant to be used.
This can become troublesome when there are a lot of patches and changes to the code.
But this is not the case here.

But your approach is fine too.